README.md

```diff
@@ -1,25 +1,32 @@
-# examples
+# Examples
 
 Examples for using Neptune to log and retrieve your ML metadata. 
 
 You can run every example with zero setup as an "ANONYMOUS" Neptune user (no registration needed).
 
 For each example you can:
-- see the script (code), 
+- see the script (<code></></code>), 
 - see rendered notebook (notebook) 
-- or open the notebook in Colab (open in colab)
+- or open the notebook in Colab (![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg))
 
 ## How to guides
 
-- Hello World [code](how-to-guides/hello-world/scripts/Neptune_hello_world.py) [notebook](how-to-guides/hello-world/notebooks/Neptune_hello_world.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/hello-world/notebooks/Neptune_hello_world.ipynb)
-- Neptune API Tour [code](./how-to-guides/how-it-works/scripts) | [notebook](how-to-guides/how-it-works/notebooks/Neptune_API_Tour.ipynb) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/how-it-works/notebooks/Neptune_API_Tour.ipynb)
-- Monitor ML runs live [code](how-to-guides/monitor-ml-runs/scripts/Monitor_ML_runs_live.py) [notebook](how-to-guides/monitor-ml-runs/notebooks/Monitor_ML_runs_live.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/monitor-ml-runs/notebooks/Monitor_ML_runs_live.ipynb)
-- Organize ML experiments [code](how-to-guides/organize-ml-experimentation/scripts/Organize_ML_runs.py) [notebook](how-to-guides/organize-ml-experimentation/notebooks/Organize_ML_runs.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/organize-ml-experimentation/notebooks/Organize_ML_runs.ipynb)
+|  | Code | Notebook | Colab |
```

Comment:
Maybe we should add a 'Description' as this first column or something like that -> not sure just a thought.

Reply:
Lets park this for the next PR.. already quite a few changes bundled into this